### ISSUE TITLE
Rate limiting plugin with Unlogged batch warning

### ISSUE THREAD

**Issue by fjsousa** [Mashape/kong#681]
I'm getting this warning when doing ab testing in a single endpoint, with the rate limit plugin enabled. 

```
WARN  11:47:11 Unlogged batch covering 6 partitions detected against table [kong.ratelimiting_metrics]. You should use a logged batch for atomicity, or asynchronous writes for performance.
```

**Comment by thibaultCha** [comment]
I am guessing you have multiple periods enabled for the same API? Like year, month, day, etc?

If this is the case then this makes sense, and the rate-limiting increasing counters should be done via independent writes rather than a simple batch. Because batching with many partitions (which happens when you have many rate-limiting periods) is inefficient.

Won't have time to fix right now but this will have to be addressed in the future. Thanks for reporting!

**Comment by fjsousa** [comment]
I had seconds and hour but then I did the same ab test with this rate-limiting setup: 
```
    "api_id": "f459f0be-e924-496a-c944-e63683207117",
    "id": "33db7fa3-8e1b-470c-c7be-645beb83eef8",
    "created_at": 1446638410000,
    "enabled": true,
    "name": "rate-limiting",
    "config": {
        "second": 5
    }

```

and got the same warning

**Comment by thibaultCha** [comment]
Yeah actually I remember no matter how many limits you have the plugin will still increment all the limits to keep track of the calls. So, 6 partitions...

**Comment by fjsousa** [comment]
got it

**Comment by thibaultCha** [comment]
This was fixed since #803 got merged. Leaving opened until the release. Thanks for reporting!
